daemon/cluster/executor/container/container.go

```diff
@@ -627,7 +627,7 @@ func (c *containerConfig) networkCreateRequest(name string) (clustertypes.Networ
 		Internal:   na.Network.Spec.Internal,
 		Attachable: na.Network.Spec.Attachable,
 		Ingress:    convert.IsIngressNetwork(na.Network),
-		EnableIPv6: na.Network.Spec.Ipv6Enabled,
+		EnableIPv6: &na.Network.Spec.Ipv6Enabled,
```

Comment:
Could a by-value copy be made? Taking the address of a persistent struct field makes me nervous.